<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnchorPopupMenu</class>
 <widget class="QWidget" name="AnchorPopupMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="layoutHorz">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnParentLeft">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/ParentLeft.png</normaloff>:/icons/anchors/ParentLeft.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelfLeft">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/SelfLeft.png</normaloff>:/icons/anchors/SelfLeft.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnParentHCenter">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/ParentHCenter.png</normaloff>:/icons/anchors/ParentHCenter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelfHCenter">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/SelfHCenter.png</normaloff>:/icons/anchors/SelfHCenter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnParentRight">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/ParentRight.png</normaloff>:/icons/anchors/ParentRight.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelfRight">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/SelfRight.png</normaloff>:/icons/anchors/SelfRight.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnParentHStretch">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/ParentHStretch.png</normaloff>:/icons/anchors/ParentHStretch.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelfHStretch">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/SelfHStretch.png</normaloff>:/icons/anchors/SelfHStretch.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutBoth">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnParentLeftTop">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>Both</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelfLeftTop">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>Both</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnParentCenter">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>Both</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelfCenter">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>Both</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnParentRightBottom">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>Both</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelfRightBottom">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>Both</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnParentStretch">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>Both</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelfStretch">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>Both</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutVert">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnParentTop">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/ParentTop.png</normaloff>:/icons/anchors/ParentTop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelfTop">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/SelfTop.png</normaloff>:/icons/anchors/SelfTop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnParentVCenter">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/ParentVCenter.png</normaloff>:/icons/anchors/ParentVCenter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelfVCenter">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/SelfVCenter.png</normaloff>:/icons/anchors/SelfVCenter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnParentBottom">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/ParentBottom.png</normaloff>:/icons/anchors/ParentBottom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelfBottom">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/SelfBottom.png</normaloff>:/icons/anchors/SelfBottom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnParentVStretch">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/ParentVStretch.png</normaloff>:/icons/anchors/ParentVStretch.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelfVStretch">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../data/Resources.qrc">
          <normaloff>:/icons/anchors/SelfVStretch.png</normaloff>:/icons/anchors/SelfVStretch.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionParentLeft">
   <property name="icon">
    <iconset resource="../../data/Resources.qrc">
     <normaloff>:/icons/anchors/ParentLeft.png</normaloff>:/icons/anchors/ParentLeft.png</iconset>
   </property>
   <property name="text">
    <string>Parent Left</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aligns horizontal anchors to the left edge of the parent&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Fixed width. Always the same pixel offset from the left.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSelfLeft">
   <property name="icon">
    <iconset resource="../../data/Resources.qrc">
     <normaloff>:/icons/anchors/SelfLeft.png</normaloff>:/icons/anchors/SelfLeft.png</iconset>
   </property>
   <property name="text">
    <string>Self Left</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aligns horizontal anchors to the left edge of the widget itself&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Fixed width. Proportional offset from the left.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionParentTop">
   <property name="icon">
    <iconset resource="../../data/Resources.qrc">
     <normaloff>:/icons/anchors/ParentTop.png</normaloff>:/icons/anchors/ParentTop.png</iconset>
   </property>
   <property name="text">
    <string>Parent Top</string>
   </property>
   <property name="toolTip">
    <string>Aligns vertical anchors to the top edge of the parent</string>
   </property>
  </action>
  <action name="actionSelfTop">
   <property name="icon">
    <iconset resource="../../data/Resources.qrc">
     <normaloff>:/icons/anchors/SelfTop.png</normaloff>:/icons/anchors/SelfTop.png</iconset>
   </property>
   <property name="text">
    <string>Self Top</string>
   </property>
   <property name="toolTip">
    <string>Aligns vertical anchors to the top edge of the widget itself</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../data/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
